.travis.yml

```diff
@@ -1,70 +1,57 @@
 # Travis can building for Linux and macOS
-matrix:
+language: python
+addons:
+  apt:
+    update: true
+    packages: libxkbcommon-x11-0  # Linux package needed for Qt to work
+  homebrew:
+    update: true
+    packages: freetype
+
+services: xvfb
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+jobs:
+  allow_failures:
+    - python: 3.8
+    - os: osx
   include:
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.5
-    - os: linux
-      dist: xenial 
-      sudo: required
-      language: python
-      python: 3.6
-    - os: linux
-      dist: xenial
-      sudo: required
-      language: python
-      python: 3.7
-    # To maximise compatibility pick earliest image, OS X 10.12
+    - python: 3.8
+      before_install: pip install pygame==2.0.0.dev6
     - os: osx
-      osx_image: xcode8.3
-      sudo: required
+      osx_image: xcode11.2
       language: generic
-      python: 3.6
+      python: 3.7
+      before_install: ln -s /usr/local/bin/pip3 /usr/local/bin/pip
```

Comment:
Could you please clarity the intent here? Why would we "write" to the test operating environment?

Reply:
This file was using pip in some places and pip3 in others.  https://snarky.ca/why-you-should-use-python-m-pip/

I am just trying to ensure in across all OSes that pip3 is the pip that is bound to the Python we are testing with.

Reply:
Thanks for the link to Brett's article: I'm very familiar with the matters.

If we were to be strict, in the sense of being certain that we're using the correct Python version, then no `pip` / `pip3` would be used.
Given that `python3 --version` is run early, letting us know which exact version is CI running then, strictly, and per Brett's article, which I do agree with, we would go `python3 -m pip` from then on... :)

What I'm wondering is why no such `ln` seems to be run under `linux`. But I'm far from a Travis expert. Feel free to instruct me, please! :)